app/views/users/index.html.erb

```diff
@@ -1,2 +1,9 @@
-<h1>Here is the list of users names, photos and post counters</h1>
-<p>This is the opening page for sure</p>
```

Comment:
[OPTIONAL] I think it's best to have a heading on the pages so that there's visible text when there are no user's created yet. Currently, the home page just looks blank until a user is created.

Reply:
Thank you will do so.